docs/interactions.rst

```diff
@@ -0,0 +1,123 @@
+
+This right here is a simple ping pong Command made with Nextcords slash feature.
+
+.. code-block:: python3
+
+    @bot.slash_command(name="ping")
+    async def ping(interaction):
+        await interaction.response.send_message("Pong!")
+        
+The way it works is that you use the slash_command function to interact with the DiscordAPI "name" is the name of your slash command, And guild_ids is used to limit the guilds that the slash command is available also useful for testing as slash commands can take up to an hour to register
+
+How To Use Sub-Commands
+-------------------------
+
+The way sub-commands work is that you will make a normal slash command that will never be called then make the sub-commands and have them do the work of real slash commands, There is no difference to slash commands and sub-commands, The Only thing you will need to change is functions. 
+
+As shown in the demistration below you make a main slash command or a Dummy slash command and build sub-commands off it
+
+.. code-block:: python3
+
+    @bot.slash_command()
+    async def main(interaction):
+        await interaction.response.send_message("This will never get called if this has subcommands.")
+
+
+    @main.subcommand()
+    async def sub1(interaction):
+        await interaction.response.send_message("This is subcommand 1!")
+
+
+    @main.subcommand(name="sub2", description="This is subcommand 2 tricked out!")
+    async def subcommand_two(interaction: Interaction,
+                            arg1: str = SlashOption(name="argument1", description="The first argument."),
+                            arg2: str = SlashOption(description="The second argument!", default=None)):
+        await interaction.response.send_message(f"This is subcommand 2 with arg1 {arg1} and arg2 {arg2}")
+        
+Fields And Requirements
+------------------------
+Fields are mean't to facilitate an easier way to fill info, Letting people doing a slash commands know what to fill in
+
+Nextcord's implementation of slash commands has fields and is very simple, In the example below is a field
+
+.. code-block:: python3
+     
+     @bot.slash_command(name="help")
+     async def help(interaction: Interaction,
+                    setting: str = SlashOption(name=settings, description="Configure Your Settings")
+         if setting == "music"
+            await interaction.response.send_message(f"MOOSIC")
+         if setting == "moderation"
+            await interaction.response.send_message(f"Mods party? POOG")
+
+
+How To Make Slash Commands In Cogs
+-----------------------------------
+Show below is an example of a simple command running in a cog, It is very basic doesn't have alot of features, Some features planned is autocomplete and proper error handling to slash commands, So that mean's that slash cogs are gonna need to get a massive upgrade and will change alot, Since this is a very simple slash command cog it won't change just the more advanced features.
```

Comment:
It is in my opinion that everything besides the first line is unnecessary.

"Shown below is an example of a simple command running in a cog."